From c3569696294bb264a0c645db12f027d44b4e033f Mon Sep 17 00:00:00 2001
From: Petr Mladek <pmladek@suse.cz>
Date: Tue, 20 Dec 2011 18:39:57 +0000
Subject: set correct icon names in desktop files

also use icon names *3.5* instead of *35*; this version is used on another
locations as well; it allows to reuse the UNIXBASISROOTNAME variable that
is substituted by the installer
---
diff --git a/sysui/desktop/productversion.mk b/sysui/desktop/productversion.mk
index cc790f8..398241d 100755
--- a/sysui/desktop/productversion.mk
+++ b/sysui/desktop/productversion.mk
@@ -63,18 +63,18 @@ PRODUCTVERSION.libreoffice = $(PRODUCTVERSION)
 PRODUCTVERSIONSHORT.libreoffice = $(PRODUCTVERSIONSHORT)
 PKGVERSION.libreoffice = $(PKGVERSION)
 UNIXFILENAME.libreoffice = $(PRODUCTNAME.libreoffice:l)$(PRODUCTVERSION.libreoffice)
-ICONPREFIX.libreoffice = $(UNIXFILENAME.libreoffice:s/.//g)
+ICONPREFIX.libreoffice = $(UNIXFILENAME.libreoffice)
 
 PRODUCTNAME.lodev = LibreOfficeDev
 PRODUCTVERSION.lodev = $(PRODUCTVERSION)
 PRODUCTVERSIONSHORT.lodev = $(PRODUCTVERSIONSHORT)
 PKGVERSION.lodev = $(PKGVERSION)
 UNIXFILENAME.lodev = lodev$(PRODUCTVERSION.lodev)
-ICONPREFIX.lodev = $(UNIXFILENAME.lodev:s/.//g)
+ICONPREFIX.lodev = $(UNIXFILENAME.lodev)
 
 PRODUCTNAME.oxygenoffice = OxygenOffice
 PRODUCTVERSION.oxygenoffice = $(PRODUCTVERSION)
 PRODUCTVERSIONSHORT.oxygenoffice = $(PRODUCTVERSIONSHORT)
 PKGVERSION.oxygenoffice = $(PKGVERSION)
 UNIXFILENAME.oxygenoffice = $(PRODUCTNAME.oxygenoffice:l)$(PRODUCTVERSION.oxygenoffice)
-ICONPREFIX.oxygenoffice = $(UNIXFILENAME.oxygenoffice:s/.//g)
+ICONPREFIX.oxygenoffice = $(UNIXFILENAME.oxygenoffice)
diff --git a/sysui/desktop/share/makefile.mk b/sysui/desktop/share/makefile.mk
index 88954ae..f6a34b1 100755
--- a/sysui/desktop/share/makefile.mk
+++ b/sysui/desktop/share/makefile.mk
@@ -150,7 +150,7 @@ $(LAUNCHERFLAGFILE) : $(LAUNCHERDEPN)
     @@-$(MKDIRHIER) $(@:db).$(INPATH).$(@:f)
     @echo Creating desktop entries for $(@:f) ..
     @echo ---------------------------------
-    @$(PERL) brand.pl -p '$${{PRODUCTNAME}} $${{PRODUCTVERSION}}' -u $(UNIXWRAPPERNAME) --iconprefix '$(ICONPREFIX.$(@:d:d:f))-' $< $(@:db).$(INPATH).$(@:f)
+    @$(PERL) brand.pl -p '$${{PRODUCTNAME}} $${{PRODUCTVERSION}}' -u $(UNIXWRAPPERNAME) --iconprefix '$${{UNIXBASISROOTNAME}}-' $< $(@:db).$(INPATH).$(@:f)
     @$(PERL) translate.pl -p '$${{PRODUCTNAME}} $${{PRODUCTVERSION}}' -d $(@:db).$(INPATH).$(@:f) --ext "desktop" --key "Name" $(ULFDIR)$/launcher_name.ulf
     @$(PERL) translate.pl -p '$${{PRODUCTNAME}} $${{PRODUCTVERSION}}' -d $(@:db).$(INPATH).$(@:f) --ext "desktop" --key "Comment" $(ULFDIR)$/launcher_comment.ulf
     @$(PERL) translate.pl -p '$${{PRODUCTNAME}} $${{PRODUCTVERSION}}' -d $(@:db).$(INPATH).$(@:f) --ext "desktop" --key "GenericName" $(ULFDIR)$/launcher_genericname.ulf
--
cgit v0.9.0.2-2-gbebe
